File: src/app/movie-response.ts
export interface MovieResponse {
    moviePK: MoviePk
    noOfTickets: number
    movieGenre: string
    movieHours: string
    movieLanguage: string
    movieDescription: string
    movieRating: string
    movieDate: any
    ticketStatus: boolean
    movieImage: MovieImage
  }
  
  export interface MoviePk {
    movieName: string
    theatreName: string
  }
  
  export interface MovieImage {
    id: string
    title: string
    image: Image
  }
  
  export interface Image {
    type: number
    data: string
  }

File: moviebookingapp/src/main/java/com/moviebookingapp/dto/MovieResponse.java
package com.moviebookingapp.dto;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDate;

@Getter
@Setter
public class MovieResponse {

    private String movieName;
    private String theatreName;
    private String movieGenre;
    private String movieHours;
    private String movieLanguage;
    private String movieDescription;
    private String movieRating;
}
